Documentation-Requirements.md

```diff
@@ -177,3 +193,28 @@ source file and re-generating the binary format image file.
 Ideally, the format of all image source files is an open standard, non-binary
 one such as SVG. Text formats are highly preferable because you can manipulate
 and compare them with standard tools (e.g. `diff(1)`).
+
+# Names
+
+Some documents need to specify the name of people. Such names should be
+written in backticks. The main reason for this is to keep the spell checker
+happy (since it cannot handle all possible names). However, since backticks
+are rendered in a fixed-width font, this makes the names clearer:
+
+> Welcome to `Clark Kent`, the newest member of the Kata Containers Architecture Committee.
+
+# Version numbers
+
+Version numbers should be written in backticks. This keeps the spell checker
+happy but since backticks are rendered in a fixed-width font, it also makes
+the numbers clearer:
```

Comment:
Lines 208-210 suggested rewrite: 
Write version number in backticks. This keeps the spell checker happy and since backticks render in a fixed-width font, it also makes the numbers clearer: